event_delegate.go

```diff
@@ -49,13 +49,16 @@ type NodeEvent struct {
 }
 
 func (c *ChannelEventDelegate) NotifyJoin(n *Node) {
-	c.Ch <- NodeEvent{NodeJoin, n}
+	node := *n
```

Comment:
Note: This is a shallow copy of `Node.Meta` which is a `[]byte`

Reply:
Ah yes, good point! This interface is still racy, so this is only a quick hack to get the tests passing the race detector. We will need to make a larger breaking change to this interface to fix it properly for serf and Consul.

Reply:
:+1: